Vinaigrette

ZeroWing opened the pull request:
Added a dressing to the salad section. Please approve.

allyourcodearebelongtous committed:
Create Vinaigrette.txt

allyourcodearebelongtous committed:
changed to markdown

allyourcodearebelongtous committed:
added pic
fixed typo

allyourcodearebelongtous committed:
added pic

ZeroWing committed:
Update Vinaigrette.md

allyourcodearebelongtous committed:
Merge branch 'vinaigrette' of https://github.com/ZeroWing/lets_git_kochbuch into vinaigrette

ancchi reviewed (changes requested):
Hi ZeroWing,

bis zur Zubereitung sieht es sehr gut aus.

Dann hast du aber zweimal die Überschrift "Zubereitung" und den ersten Satz drin und bei den Schritten 1 und 2 sind diverse (Schussel-) Fehler drin.
Vorschlag:
Den Satz "Bei Blattsalaten sollte die Vinaigrette erst kurz vor dem Servieren beigemischt werden, da der Salat sonst schnell "lädschig" wird, also nicht mehr so bissfest ist." als Tipp HINTER der Zubereitung mit der Überschrift "Tipp" hinzufügen.
Ansonsten den Satz natürlich bei Zubereitung rausnehmen und bei den Punkten 1 und 2 die Fehler korrigieren.
Viel Erfolg! ^^

ZeroWing committed:
Pull Request Kommentare beherzigt

Die Anmerkungen des Pull Request Reviewer umgesetzt (die Fehler waren natürlich absichtlich eingebaut) und um eine Zubereitungsvariante ergänzt

ZeroWing committed:
hoppla, zu schnell

jetzt aber...

ZeroWing committed:
aufgehübscht

2 ist zwei

ancchi committed:
Komma

Habe nur noch ein Komma zwischen Pfeffer und Zucker hinzugefügt, ansonsten ist jetzt alles schick. :)
